R/smtp_send.R

```diff
@@ -92,73 +119,99 @@ smtp_send <- function(email,
     }
   }
 
+  # If the user provides a path to a creds file in the `creds_file`
+  # argument, upgrade that through the `creds_file()` helper function
+  # and provide a warning about soft deprecation
+  if (!missing(creds_file)) {
+    credentials <- creds_file(creds_file)
+    warning("The `creds_file` argument is undergoing soft deprecation:\n",
```

Comment:
s/undergoing soft deprecation/deprecated/

Reply:
This change was made. Thanks!